kahuna/public/js/search/query.html

```diff
@@ -55,6 +56,17 @@
             </li>
 
             <li class="search__filter-item">
+                <label class="search__filter">
+                    <input type="checkbox"
+                           ng:model="searchQuery.ordering.orderBy"
+                           ng:false-value="undefined"
+                           ng:true-value="'oldest'"
+                           ng:checked="searchQuery.ordering.orderBy === 'oldest'" />
+		    Oldest first
```

Comment:
I think it should be in the search bar, but perhaps it should be obviously distinct. Perhaps some kind of separator between this and the filter options?